frontend/src/components/HomeTopBar/HomeTopBar.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button, AppBar, Toolbar, withStyles, makeStyles } from '@material-ui/core';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import styles from './HomeTopBar.module.css';
+// import * as PropTypes from "prop-types";
+
+// TODO Flags for host
```

Comment:
hmm not sure why this commenting is still here, this was removed in a PR from a while ago. Maybe it was still in your local and it stayed there through the merge? neway pls delete <3